nall/terminal.hpp

```diff
@@ -4,59 +4,56 @@
 
 namespace nall::terminal {
 
-inline auto escapable() -> bool {
-  #if defined(PLATFORM_WINDOWS)
-  //todo: colors are supported by Windows 10+ and with alternate terminals (eg msys)
-  //disabled for now for compatibility with Windows 7 and 8.1's cmd.exe
-  return false;
-  #endif
-  return true;
-}
+#if defined(PLATFORM_WINDOWS)
+bool escape_colors = false;
```

Comment:
Perhaps this should be renamed something like "enable_colors" or "use_color", making it's purpose more clear

Reply:
Thanks for the feedback. I think I will rework this even more and include true color support as well ..